### ISSUE TITLE
Outline view support

### ISSUE THREAD

**Issue by coffenbacher** [rubyide/vscode-ruby#369]
The Outline View is [now out of preview and in mainstream VSCode](https://code.visualstudio.com/updates/v1_25#_outline-view)

Is this extension where support for that would go? Is this on the roadmap? It would be a super useful feature for me.

Thanks!

**Comment by wingrunr21** [comment]
I believe this will be supported once we're reporting full document symbols back. Once I work through the `tree-sitter` stuff and can merge #366 then the next step on my plan is to work the linters and symbol support (which enables stuff like this and go to definition).

**Comment by theblang** [comment]
@coffenbacher @wingrunr21 I'm seeing that #366 is merged and it appears that the outline view does work for ruby files! I'm not seeing it work for rspec files though, which would be super helpful to me. What are your thoughts?

**Comment by wingrunr21** [comment]
#405 adds outline view not #366

**Comment by fuzzy76** [comment]
For some files, I see outline content for half of the file...

**Comment by wingrunr21** [comment]
Outlline view hasn't shipped yet so not sure how that's happening

**Comment by fuzzy76** [comment]
Scratch that, I do see it all. I just needed to adjust the sorting to realize I saw everything. Using 0.21.1 on vscode 1.31.1. This is the only Ruby extension I have installed.

I seem to remember it started showing up after installing all of the extras mentioned in the readme (rubocop, rcodetools, debug support, etc).

**Comment by wingrunr21** [comment]
It must be one of the older integrations providing legacy symbols for the outline view. Would you mind posting a screenshot? I don't have that in my VSCode installs and I don't want the outline view support in #405 to potentially regress.

**Comment by fuzzy76** [comment]
Sorry for the slowness. Apparently I got the release. Now I have two trees, one more detailed than the other.

![ruby symbols](https://user-images.githubusercontent.com/218995/53232989-94b93d80-368c-11e9-8cf3-058b3fca149e.png)

**Comment by wingrunr21** [comment]
Are you using rubyLocate?

**Comment by fuzzy76** [comment]
Yes. My configuration is rcodetools for code completion, rubyLocate for intellisense, no bunder, language server enabled. And I installed just about every package mentioned in the readme. :)

Probably a bit late disclaimer: I'm not normally a Ruby developer, but I just inherited a couple of Ruby projects at my employer. So I only recently set up vscode and my local machine for this.

**Comment by wingrunr21** [comment]
Ok great. One of those tools must be sending over the data for VSCode to populate the outline view. I’ll take a look.
